IND/MarketIndices/BasketIndices.rdf

```diff
@@ -103,8 +104,8 @@
 			</owl:Restriction>
 		</rdfs:subClassOf>
 		<rdfs:label xml:lang="en">basket of credit risks</rdfs:label>
-		<skos:definition xml:lang="en">basket of securities and / or legal entities, collected for the purpose of identifying risk</skos:definition>
-		<skos:editorialNote xml:lang="en">From OTPP: Variety: Criteria for constituent entities is based on sectors (emerging market, financial, sovereign, etc), spread range (investment grade, non-investment grade), or asset type (loan, bond, mortgage-backed, asset-backed), second criteria is based on maturity of protection (2,3,5,7,10 yrs). As a result, I&apos;m aware of over 2000 CDS indexes that Markit manages. Constituents can be on multiple indexes, based on maturity, etc. Because the indexes are maturity specific, it makes sense to have an entity AND an instrument with the corresponding maturity (or close to it) to measure the credit-risk of default. Premiums are standard and set depending on the composite spread at the initiation of the index, with upfront-fees to get into a deal thereafter depending on changes in the MTM of the spread. The payout per constituent credit-event is ~ Notional x weight. Modeling notes: Whereas this may be a basket of securities just like a &quot;Basket&quot; in the sense used in reference indices and in derivatives with basket underlyers, in this instance the constituents of the basket are collected, and are defined as constituents, in the context of the credit risks that they represent and not in the context of their market price as securities.</skos:editorialNote>
+		<skos:definition xml:lang="en">basket of instruments and legal entities collected for the purpose of identifying risk</skos:definition>
```

Comment:
Generally will contain one or the other but not both. And certainly not required to contain both which is what "and" implies. 
Also "analyzing" would be better than "identifying"

Reply:
Changed and to 'and/or', which, of course, violates your ambiguity issue, but is the correct definition.  For an instrument-specific risk, the risk is calculated based on both the risk associated with a given instrument AND the risk associated with the entity overall